# Add keywords to desktop file.

diff --git a/data/org.kde.kdenlive.desktop b/data/org.kde.kdenlive.desktop
--- data/org.kde.kdenlive.desktop
+++ data/org.kde.kdenlive.desktop
@@ -126,7 +126,7 @@ X-DocPath[cs]=http://cs.wikibooks.org/wiki/Kdenlive
 Terminal=false
 MimeType=application/x-kdenlive;
 Categories=Qt;KDE;AudioVideo;AudioVideoEditing;
-Keywords=editing;video;audio;mlt;kde;
+Keywords=video;audio;firewire;video4linux;suite;editing;kde;mlt;
 Keywords[ca]=edició;vídeo;àudio;mlt;kde;
 Keywords[ca@valencia]=edició;vídeo;àudio;mlt;kde;
 Keywords[da]=redigering;video;lyd;mlt;kde;
