# Add keywords to desktop file.

diff -Naur kdenlive-0.9.6.orig/data/kdenlive.desktop kdenlive-0.9.6/data/kdenlive.desktop
--- kdenlive-0.9.6.orig/data/kdenlive.desktop	2013-04-06 21:25:09.000000000 +0200
+++ kdenlive-0.9.6/data/kdenlive.desktop	2013-08-13 11:04:33.996558347 +0200
@@ -105,4 +105,5 @@
 X-DocPath[cs]=http://cs.wikibooks.org/wiki/Kdenlive
 Terminal=false
 MimeType=application/x-kdenlive;
+Keywords=video;audio;firewire;video4linux;suite;editing;kde;mlt;
 Categories=Qt;KDE;AudioVideo;AudioVideoEditing;
